src/test/java/HerokuappTestNG.java

```diff
@@ -1,61 +1,92 @@
-import org.openqa.selenium.Alert;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
 import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
 import java.util.List;
 
 public class HerokuappTestNG {
 
+    WebDriver driver;
+
+    @BeforeTest
+    public void setup() {
+        System.setProperty("webdriver.chrome.driver", "C:\\WebDrivers\\chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.get("https://the-internet.herokuapp.com/");
+        driver.manage().window().maximize();
+    }
+
     @Test
-    public void TestNG() {
-        System.setProperty("webdriver.chrome.driver",
-                "C:\\WebDrivers\\chromedriver.exe");
-        WebDriver ChromeForHomework1 = new ChromeDriver();
-        ChromeForHomework1.get("https://the-internet.herokuapp.com/");
-        List<WebElement> links = ChromeForHomework1.findElements(By.tagName("a"));
+    public void clickAndAlertsHomework() {
```

Comment:
These should be 4 different tests. One for click, and three for all alert types

Reply:
I can't understand, why after spliting it to separate methods, only the first one works. It looks like, that driver is not closed after each @Test, and correspondingly elements can not be found, because the page from the previous method is open